### PR TITLE
feat: Control creation of SAML configuration via `create_saml_configuration`

### PR DESCRIPTION
**Pull request by csantanapr**
## Description
Allow to skip saml configuration

## Motivation and Context
Related to #19 

## Breaking Changes
No breaking changes

## How Has This Been Tested?
- [x] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [x] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [x] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -182,6 +182,12 @@ variable "iam_role_tags" {
 # Workspace SAML Configuration
 ################################################################################
 
+variable "create_saml" {
```
Comment (bryantbiggs) [inline variables.tf:185]: Looks great - lets use the slightly longer form just to be explicit 
```suggestion
variable "create_saml_configuration" {
```

Comment (csantanapr) [reply variables.tf:185]: Sounds good @bryantbiggs 👍

**Review by bryantbiggs — APPROVED** [review]
thank you!

**Comment by antonbabenko** [comment]
This PR is included in version 1.9.0 :tada:

**Pull request merged**